travis/script/predicate_functions.sh

```diff
@@ -86,7 +98,11 @@ function additional_specs_available {
 function documentation_enforced {
```

Comment:
@pirj @benoittgt we need to figure out why yard isn't running on 2.7.0, but suggest we merge this to indicate the state of our other builds then figure out how to restore this check

Reply:
Let's do it 👍